app/Http/Livewire/User/EventComponent.php

```diff
@@ -64,18 +65,25 @@ private function resetInput()
         $this->cover_path = null;
         $this->location = null;
         $this->date_event = null;
+        $this->date_time = null;
         $this->description = null;
         $this->tags = null;
+        $this->format = 'd.m.Y';
```

Comment:
This line has date format different from everywhere else

Reply:
This is assigning a standard value before receiving JS data from the browser